zenodo/modules/spam/views.py

```diff
@@ -144,8 +162,13 @@ def _expand_users_info(results, include_pending=False):
         ).filter(User.id.in_(results.keys()))
     )
 
+    normalized_emails_list = Counter([
```

Comment:
```suggestion
    normalized_emails = Counter([
```
Since it's not a list 😏